conda_build/license_family.py

```diff
@@ -14,11 +13,11 @@
 GPL
 BSD
 MIT
-APACHE
+Apache
 PSF
-PUBLICDOMAIN
-PROPRIETARY
-OTHER
+Public-Domain
```

Comment:
Also another point on the `license_family`. Currently `conda skeleton pypi` seems to be filling out this information. If people shouldn't be setting it, maybe that tool should stop adding it.

Though better yet would be to get `conda-build` to issue some sort of warning that the license family field is deprecated and subject to removal.